app/queries/consumer_apps/find_or_create_by_query.rb

```diff
@@ -0,0 +1,28 @@
+module ConsumerApps
+  class FindOrCreateByQuery
+    def self.call(app_bundle:, platform:)
+      new(app_bundle: app_bundle, platform: platform).call
+    end
+
+    def initialize(app_bundle:, platform:)
+      @app_bundle = app_bundle
+      @platform = platform
+      @forem_bundle = ConsumerApp::FOREM_BUNDLE
```

Comment:
I would keep this as as constant, as by assigning it to an instance variable it somehow implies it could be changed in the lifetime of the query